libs/designsystem/src/lib/components/modal/alert/alert.component.ts

```diff
@@ -36,10 +34,13 @@ export class AlertComponent implements AfterViewInit {
     return this._ionPageReset;
   }
 
-  constructor(private modalController: IModalController) {}
+  constructor(private elementRef: ElementRef<HTMLElement>) {}
 
   ngAfterViewInit(): void {
-    this.modalController.blurNativeWrapper(this.alertWrapper.nativeElement);
+    setTimeout(() => {
+      this.alertWrapper.nativeElement.focus();
+      this.alertWrapper.nativeElement.blur();
+    }, 50);
```

Comment:
Consider extracting `50` to a variable, to have it be a "bit less magical"

Reply:
I didn't create or change this, simply inlined an existing method - but OK ;)